src/main/java/zmq/Ctx.java

```diff
@@ -127,6 +134,13 @@ private void destroy() throws IOException
             it.close();
         }
 
+        for (Selector selector : selectors) {
+            if (selector != null) {
```

Comment:
That thought occurred to me as well... do you think it's worth leaving the null check in there, just in case? I don't know the full context, but I haven't seen any code that suggests a selector in the list might be null.